x/wasm/internal/keeper/keeper_test.go

```diff
@@ -1025,6 +1025,66 @@ func TestMigrateWithDispatchedMessage(t *testing.T) {
 	assert.Equal(t, deposit, balance)
 }
 
+type sudoMsg struct {
+	StealFunds stealFundsMsg `json:"steal_funds"`
+}
+
+type stealFundsMsg struct {
+	Recipient string            `json:"recipient"`
+	Amount    wasmvmtypes.Coins `json:"amount"`
+}
+
+func TestSudo(t *testing.T) {
+	ctx, keepers := CreateTestInput(t, false, SupportedFeatures, nil, nil)
+	accKeeper, keeper, bankKeeper := keepers.AccountKeeper, keepers.WasmKeeper, keepers.BankKeeper
+
+	deposit := sdk.NewCoins(sdk.NewInt64Coin("denom", 100000))
+	creator := createFakeFundedAccount(t, ctx, accKeeper, bankKeeper, deposit.Add(deposit...))
+
+	wasmCode, err := ioutil.ReadFile("./testdata/hackatom.wasm")
+	require.NoError(t, err)
+	contractID, err := keeper.Create(ctx, creator, wasmCode, "", "", nil)
+	require.NoError(t, err)
+
+	_, _, bob := keyPubAddr()
+	_, _, fred := keyPubAddr()
+	initMsg := HackatomExampleInitMsg{
+		Verifier:    fred,
+		Beneficiary: bob,
+	}
+	initMsgBz, err := json.Marshal(initMsg)
+	require.NoError(t, err)
+
+	addr, _, err := keeper.Instantiate(ctx, contractID, creator, nil, initMsgBz, "demo contract 3", deposit)
+	require.NoError(t, err)
+	require.Equal(t, "cosmos18vd8fpwxzck93qlwghaj6arh4p7c5n89uzcee5", addr.String())
+
+	// the community is broke
+	_, _, community := keyPubAddr()
+	comAcct := accKeeper.GetAccount(ctx, community)
+	require.Nil(t, comAcct)
+
+	// now the community wants to get paid via sudo
+	msg := sudoMsg{
+		StealFunds: stealFundsMsg{
```

Comment:
Not sure if this is the best use case of this function 🤣 

👮👮👮 But more serious, people tend to not understand jokes very well when it comes to money or security. Maybe some additional context can help tot explain that this is not about stealing money but calling a privileged function in the contract that would otherwise not be available. This example would not work with random contract but only with the hackatom example that had this `stealFundsMsg` function added for this particular demo case. We do not encourage people to build backdoors into their contracts... 👮 👮👮

🤣 🤣 🤣

Reply:
I will add a comment. I added it in CosmWasm as a funny example.
Our first tutorial of cosmwasm was to modify the escrow contract to add a backdoor. And then said this is why it is essential you verify the actual rust code.

Reply:
Rebased and added a comment in https://github.com/CosmWasm/wasmd/pull/433/commits/311be37e90f8932d8754f53612732e87f9bb18ba